### ISSUE TITLE
HTTPS Insecure option required

### ISSUE THREAD

**Issue by ghost** [gotsunami/gitlab-copy#30]
If the gitlab instance uses a company CA-signed certificate (which are often not then signed by a global CA) then certificate cannot be validated and the process fails

Can you add a option type to the config.yml to skip verification? Kind of equivalent to wget's no-check-certificate flag

Output when such a case arises looks like:

```
DUMMY MODE: won't apply anything (stats only)
--
2018/05/30 21:19:38 network error: Get https:/api/v3/projects/namespace%2Fproject: x509: certificate signed by unknown authority
```

**Comment by matm** [comment]
I don't agree. The current behaviour is correct. With free services like Let's Encrypt around, no more company should have self-signed certificates.

**Comment by erzz** [comment]
Absolutely! But this is not self-signed we are talking about. Company
INTERNAL CA's are a completely different matter and they are completely
trusted within the scope that they are used and for several reasons they
are often isolated from the wider CA scope on internet.

**Comment by matm** [comment]
Okay, I better understand your use case, thanks. WDTY @aboutofpluto ?

**Comment by aboutofpluto** [comment]
Hi erzz, hi matm.
That makes sense. But that is actually quite rare I think.
We could add an option with a warning message.
But I am not sure we can detect the difference between an internal CA and a wider not-signed CA.

**Comment by erzz** [comment]
I don't think that's what you necessarily need to do. Whatever library you
use to establish the connection, they usually have an option /flag
/parameter. --no-check-certificate... -insecure... -no-verify... Something
like that. Connect an option in your configuration file to execute with the
flag

**Comment by matm** [comment]
@aboutofpluto We could go for a `-k` (or `--insecure`) flag, like `curl`.

**Comment by aboutofpluto** [comment]
OK for `-k` and `--insecure`!****

**Comment by matm** [comment]
Fixed by #55.
